crates/testing/tests/vid_task.rs

```diff
@@ -48,7 +51,7 @@ async fn test_vid_task() {
         commitment: block_commitment,
     };
 
-    let signature = committee_exchange.sign_da_proposal(&block.commit());
+    let signature = BLSPubKey::sign(priv_key, block.commit().as_ref());
```

Comment:
We seem to be missing a `sign_vid_proposal` method analogous to `sign_da_proposal`. Perhaps that's because I didn't make such a method in a previous PR?

We probably should not be hard-coding the key type `BLSPubKey` here. Probably should just make a new `sign_vid_proposal` method in the `VIDExchangeType` trait.

These `sign_xxx_proposal` methods really just sign a block commitment though. Seems silly to have separately named methods that all do the same thing. A better solution would be just a generic `sign_block_commitment` method shared among different exchange types. But I'm not going to insist on that for this PR.